huaweicloud/services/vod/resource_huaweicloud_vod_media_category.go

```diff
@@ -0,0 +1,241 @@
+package vod
+
+import (
+	"context"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	vod "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/vod/v1/model"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var Children = schema.Schema{
+	Type:     schema.TypeList,
+	Computed: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"children": {
```

Comment:
children seems to be a loop nested attribute, only 1 level supported?

Reply:
at most 3 levels of category supported, so if it's a first level category, there can be 2 levels of children. If it a second or third level category, the levels of children will be less.

Reply:
we only list it's children here, not include children's children, right?

Reply:
change to json string, in case the api change to support more children level